tools/rewriter/GenCallAsm.cpp

```diff
@@ -974,6 +970,21 @@ std::string emit_asm_wrapper(AbiSignature sig,
   llvm::errs() << "Generating wrapper for " << sig_string(sig, target_name) << "\n";
   auto rets = allocate_return_locations(sig, arch);
 
+  size_t indirect_arg_size = 0;
+  for (auto &arg : args) {
+    if (arg.is_indirect()) { // Only ever true on AArch64 for now.
```

Comment:
Maybe let's assert that arch is aarch64 in this branch then?

Reply:
Done.